applications/smart-distancing/smart_distancing/core/__init__.py

```
@@ -0,0 +1,33 @@
+# common core imports
+from smart_distancing.core._centroid_object_tracker import *
+from smart_distancing.core._config_engine import *
+
+# import Distancing base class
+from smart_distancing.core._distancing import *
+
+# import Distancing implementations
```

I suggest we strictly follow the user config and if exceptions are raised terminate the program instead of falling back to a different backend. The main reason is that this is not intended as production code and we want users to have more transparency and control over the execution. Comments?

---

Sounds like a good idea to me. A bunch of code can be taken out in this case. So, in the case of a package, do we want to bundle example configs with the package and install them on first run, bundle and not install, or not bundle them at all and just list where some can be downloaded in the documentation?